src/main/java/org/embulk/output/hdfs/HdfsFileOutputPlugin.java

```diff
@@ -73,27 +69,78 @@
         @Config("delete_in_advance")
         @ConfigDefault("\"NONE\"")
         DeleteInAdvancePolicy getDeleteInAdvance();
+
+        @Config("atomic")
```

Comment:
Conventionally, embulk output plugin uses `mode: replace` to do transactional replacing.
https://github.com/embulk/embulk-output-jdbc/tree/master/embulk-output-postgresql#modes.
Why don't you follow the convention?

Reply:
Thanks, I follow the convention 8e8de7e